meteor/client/ui/SegmentTimeline/Renderers/VTSourceRenderer.tsx

```diff
@@ -561,10 +488,18 @@ export class VTSourceRendererBase extends CustomLayerItemRenderer<IProps & WithT
 					Math.abs(
 						(this.props.piece.renderedInPoint || 0) +
 							(vtContent.sourceDuration - seek) -
-							(this.props.partExpectedDuration || 0)
+							(this.props.part.instance.part.expectedDuration || 0)
 					) > 500))
 		) {
-			const endOfContentAt = (this.props.piece.renderedInPoint || 0) + (vtContent.sourceDuration - seek)
+			const lastFreeze = this.state.freezes && this.state.freezes[this.state.freezes.length - 1]
+			const endingFreezeStart =
+				lastFreeze &&
+				lastFreeze.start >= vtContent.sourceDuration &&
+				lastFreeze.start < vtContent.sourceDuration + (vtContent.postrollDuration || 0) &&
+				lastFreeze.start
+			const endOfContentAt =
+				(this.props.piece.renderedInPoint || 0) +
```

Comment:
This is a cool improvement, though some explanation would be great here, and maybe it should be optional somehow? Whether the countdown to freeze-frame should count down to actual freeze-frames or just the end of the clip is I think editorially-dependent (say, in an obit clip ending with a black-and-white photo of the deceased).

Reply:
Agreed, this should be an opt-in.
We're disabling this one for now, to be re-enabled by tv2 later and put behind a settings-flag, or blueprints property or something.